### DISCUSSION TITLE
Why mount the udev path into the pod container?

### DISCUSSION THREAD

**Discussion by microyahoo** [rook/rook#14279] [Q&A] [answered]
When creating an OSD pod, why is it necessary to mount the udev path to the pod container? What would be the impact if this is not done?
https://github.com/rook/rook/blob/f5ca09e38582d8658491e620e302c174ea11020f/pkg/operator/ceph/cluster/osd/volumes.go#L177-L188

**Comment by travisn** [comment, marked as answer]
Does this comment here answer the question?
https://github.com/rook/rook/blob/f5ca09e38582d8658491e620e302c174ea11020f/pkg/operator/ceph/cluster/osd/spec.go#L455-L456

Curious if you want to experiment with removing it, but the OSD currently requires it.

**Reply by microyahoo** [reply]
Yes, I'm also curious if this is necessary and what the effect would be after removing it. :smiley:
